### PR TITLE
fix broken link on admin flags page

### PR DESCRIPTION
**Pull request by Tenacious-Qi**
This pull request addresses this [issue](https://github.com/TheOdinProject/theodinproject/issues/1840).

After this commit, as outlined very helpfully by @CouchofTomato,  clicking on the lesson link from the admin flags page now opens the actual lesson page, instead of a 404 page.

### CODE REVIEW

**Comment by CouchofTomato** [comment]
@Tenacious-Qi Unfortunately mate I can't take any credit. Kevin edited my post to write all the good stuff 😆

File: app/admin/flags.rb [hunk 1]
```diff
@@ -32,7 +32,10 @@
         flag.project_submission.user.username
       end
       row :lesson do
-        link_to flag.project_submission.lesson.title, flag.project_submission.lesson, target: '_blank'
+        link_to flag.project_submission.lesson.title, path_course_lesson_path(flag.project_submission.lesson.path,
```
Comment (KevinMulhern) [inline app/admin/flags.rb:35]: Just a nitpick, can we make the arguments multiline for this instead please?

```
link_to( 
  flag.project_submission.lesson.title,
  path_course_lesson_path(
    flag.project_submission.lesson.path,
    flag.project_submission.lesson.course,
    flag.project_submission.lesson
  ),
  target: '_blank'
)
```

Sorry, Rubocop probably insisted on indenting it. It's so much more readable and atheistical pleasing™️  when its multiline though 😆

**Review by KevinMulhern — COMMENTED** [review]
Thanks for taking the time to do this @Tenacious-Qi 💪 

Just one very minor comment on formatting.

**Comment by Tenacious-Qi** [comment]
Should be good now. Looking forward to contributing again soon :smile:

**Review by KevinMulhern — APPROVED** [review]
LGTM 👍 
Thanks for fixing this @Tenacious-Qi 💪

**Pull request merged**